scripts/ec2cleanup.py

```diff
@@ -81,11 +82,14 @@ def delete_unattached_volumes(provider_mgmt, excluded_volumes, output):
     try:
         for volume in provider_mgmt.get_all_unattached_volumes():
             if excluded_volumes and volume.id in excluded_volumes:
-                logger.info("  Excluding volume id: %r", volume.id)
+                logger.info("  Excluding volume id: %r", volume["VolumeId"])
                 continue
             else:
-                volume_list.append([provider_name, volume.id])
-                volume.delete()
+                if datetime.now(pytz.utc) - volume["CreateTime"] > timedelta(hours=3):
```

Comment:
It it not possible to use `datetime.utcnow()`? really we are facing `daylight savings` problem in this manipulation?

Reply:
I had it there, but AWS output is always offset-aware and then I get Exception:
`TypeError: can't subtract offset-naive and offset-aware datetimes`